backend/package.json

```diff
@@ -8,6 +8,7 @@
   "main": "src/app.js",
   "scripts": {
     "ci": "npm run lint && npm test",
+    "kinesis": "node bin/kinesis.js",
```

Comment:
Yeah at the moment it's just a thing you would run in a separate terminal, just like you have to have postgres running before you do anything. You're right, it should be documented.

Hopefully that will go away very soon, as we get #129 happening.